### PR TITLE
Add CSS folder and styling

### PR DESCRIPTION
**Pull request by Vicentefg10**
Please let me know your feedback, I will schedule a 1:1.

### CODE REVIEW

**Comment by mannsn** [comment]
Check your pull request - I don't see your updates.

**Review by mannsn — APPROVED** [review]
Good job - see comments on file. I will need you to resubmit assignment so I can give you credit, but you can merge.

From AI reviewer:
Your work on implementing the specified CSS elements is commendable. You've achieved a consistent color scheme, adjusted font sizes, weights, and colors for headings, and provided a centric alignment for content within sections. The hover effects on Connect section links also enhance user interaction beautifully. You're well on your way to refining your design even further with these changes.

File: css/index.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  text-align: center;
+}
+h1,
+h2,
+h3 {
+  font-size: 2em;
+  font-weight: bold;
+  color: blueviolet;
+}
+h1#name {
```
Comment (mannsn) [inline css/index.css:18]: Does this style a section?

**Review by mannsn — APPROVED** [review]

File: css/index.css (hunk 1 above)
Comment (Vicentefg10) [reply css/index.css:18]: It targets the h1 element with the ID "name" (Vicente Flores).

**Pull request merged**